src/tv2_afvd_studio/getBaseline.ts

```diff
@@ -59,6 +59,11 @@ export function getBaseline(coreContext: IStudioContext): BlueprintResultBaselin
 		}
 	}
 
+	const defaultInput =
```

Comment:
Would be nice to specify the type of `defaultInput` as it is not clear what type the right hand side returns. What type is `context.config.studio.SwitcherSource.Default`?